app/Http/Controllers/Admin/DigitalPublicationArticleController.php

```diff
@@ -62,12 +72,43 @@ protected function indexData($request)
         ];
     }
 
+    protected function transformIndexItems($items)
+    {
+        // If we're in the browser, don't transform the items
+        if (property_exists($items, 'path')) {
+            return $items;
+        }
+        return $items->toTree();
```

Comment:
Change this to `parent::transformIndexItems($items)`, so we call the parent method in case it's changed in Twill udpates.